<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File Information Viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <!-- 文件信息表格 -->
   <widget class="QTableWidget" name="file_table">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>500</width>
      <height>400</height>
     </rect>
    </property>
   </widget>
   <!-- 文件预览区域 -->
   <widget class="QLabel" name="file_preview">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>20</y>
      <width>240</width>
      <height>180</height>
     </rect>
    </property>
    <property name="text">
     <string>Preview Area</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <!-- 操作按钮 -->
   <widget class="QPushButton" name="button_load_file">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Load File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_delete_file">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>450</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_reload_file">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>450</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Reload File</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
